zkevm-circuits/src/state_circuit/lexicographic_ordering.rs

```diff
@@ -0,0 +1,278 @@
+use super::{N_LIMBS_ACCOUNT_ADDRESS, N_LIMBS_ID, N_LIMBS_RW_COUNTER};
+use crate::{
+    evm_circuit::{param::N_BYTES_WORD, witness::Rw},
+    gadget::is_zero::{IsZeroChip, IsZeroConfig, IsZeroInstruction},
+    util::Expr,
+};
+use eth_types::{Field, ToBigEndian};
+use halo2_proofs::{
+    circuit::Region,
+    plonk::{Advice, Column, ConstraintSystem, Error, Expression, Fixed, VirtualCells},
+    poly::Rotation,
+};
+use itertools::Itertools;
+use std::ops::Mul;
+
+// 2 limbs for tag, field_tag and id.
+// 10 limbs for address,
+// 16 limbs for storage key
+// 2 limbs for rw_counter
```

Comment:
not sure why need limbs for `rw_counter`

Reply:
rw_counter is 32 bits, which is two 16 bit limbs.

Reply:
yes, the `F` type is enough,  you can compare it directly, no need to compare limbs in 32 bytes type.